source/content/guides/autopilot/07-troubleshoot.md

```diff
@@ -404,6 +404,22 @@ Update the database on the target environment, and then retry Autopilot. You wil
   
 </Accordion>
 
+## Upstreams Could Not Be Merged
+
+<Accordion title="Autopilot was unable to apply upstream updates because the upstreams could not be merged." id="upstream-no-common-ancestor" icon="info-sign">
```

Comment:
```suggestion
<Accordion title="We could not apply upstream updates because the upstreams could not be merged." id="upstream-no-common-ancestor" icon="info-sign">
```